jQuery/ajax.js

```diff
@@ -1,82 +1,113 @@
-function ajax(citizens) {
-  var characters = getUniqueValuesForField('character', citizens);
-  var hometowns = getUniqueValuesForField('hometown', citizens);
-  var sizes = getUniqueValuesForField('size', citizens);
-
-  populateSelectWithOptions("#character-select", characters);
-  populateSelectWithOptions("#hometown-select", hometowns);
-  populateSelectWithOptions("#size-select", sizes);
-
-  var selectHandler = function(event) {
-    var params = gatherParams();
-    var filteredCitizens = filter(params, event.data.citizens);
-    redrawContent(filteredCitizens);
+window.hh = {};
+
+window.hh.selectors = {
+  menu: '.menu',
+  content: '.content',
+  nameInput: '#name-input',
+  characterSelect: '#character-select',
```

Comment:
Also, if these selectors are only being used in a single function, it's an acceptable practice to put them as a variable at the top of the module where they make an appearance.